docs/1.10/working-with-projects/project-configuration/implementing-content-trust.md

```diff
@@ -19,10 +19,16 @@ The root key is generated at: `/root/.docker/trust/private/root_keys`
 You will also be asked to enter a new passphrase for the image. This is generated at `/root/.docker/trust/private/tuf_keys/[registry name] /[imagepath]`.
 If you are using a self-signed cert, make sure to copy the CA cert into `/etc/docker/certs.d/10.117.169.182` and `$HOME/.docker/tls/10.117.169.182:4443/`. When an image is signed, it is indicated in the Web UI.
 
+When an image is signed, it has a tick shown in UI; otherwise, a cross sign(X) is displayed instead.
```

Comment:
A signed image will have a checkbox next to it, otherwise an X is displayed instead